core/state/node/schema.go

```diff
@@ -63,12 +63,11 @@ func (s NodeState) Bytes() ([]byte, error) {
 }
 
 func (s NodeState) GetAppByID(appID string) (*AppInstallationState, error) {
```

Comment:
nit: I think the convention is to name this just `(s State)` so that other packages can say `node.State` rather than `node.NodeState` which is redundant

Reply:
True, we can do this in a later PR since this one is pretty big already